BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -337,12 +338,19 @@ - (void)presentInAppAlert:(NSDictionary *)userInfo {
     [self trackPushViewedWithParameters:userInfo];
     self.userInfo = userInfo;
     // Handle push notification when the app is in active state...
-    UIViewController *topViewController = [self topViewController:[[UIApplication sharedApplication].keyWindow rootViewController]];
-    BlueShiftAlertView *pushNotificationAlertView = [[BlueShiftAlertView alloc] init];
-    pushNotificationAlertView.alertControllerDelegate = (id<BlueShiftAlertControllerDelegate>)self;
-    if (@available(iOS 8.0, *)) {
-        UIAlertController *blueShiftAlertViewController = [pushNotificationAlertView alertViewWithPushDetailsDictionary:userInfo];
-        [topViewController presentViewController:blueShiftAlertViewController animated:YES completion:nil];
+    @try {
+        if ([BlueShiftInAppNotificationHelper checkAppDelegateWindowPresent] == NO) {
```

Comment:
can we replace this with a positive statement (== YES) and bring in the below code inside this if block? will save one line of code :)

Reply:
Fixed.